### DISCUSSION TITLE
Fine-tuning on multi GPUs

### DISCUSSION THREAD

**Discussion by HenryYuen128** [openai/whisper#797] [Q&A]
I got this error while fine-tuning on multi GPUs (A5000), but it works on single GPU.
![image](https://user-images.githubusercontent.com/33191354/210232965-0f82ae93-5ac8-427a-9e73-789c7ab0115b.png)

**Comment by spacemonqi** [comment]
Hi, can you share the code you used to make it run on multiple GPUs? Otherwise it's rather hard to debug...

**Reply by silvacarl2** [reply]
you may have to disable wandb

what is the command you are using to take advnatge of multiple GPUs?
